src/pilet-build.test.ts

```diff
@@ -214,4 +214,144 @@ runTests('pilet-build', ({ test, setup }) => {
       });
     },
   );
+
+  // 'can build a pilet with default target ./dist',
+  test(
+    'from-sample-piral-with-default-target',
+    'can build a pilet with default target ./dist/index.js',
+    ['pilet.v2', 'build.pilet'],
+    async (ctx) => {
+      await ctx.run(`npx pilet build --target`);
+
+      await ctx.assertFiles({
+        'dist/index.js'(content: string) {
+          expect(content).not.toBe('');
+          expect(content).toContain('//@pilet v:2');
+          expect(content).toContain('System.register(');
+          expect(content).not.toContain('currentScript.app=');
+          expect(content).not.toContain('require("react")');
+        },
+      });
+    },
+  );
+
+  // 'from-sample-piral-with-different-target' --done,
+  test(
+    'from-sample-piral-with-different-target',
+    'can build a pilet with a different target',
+    ['pilet.v2', 'build.pilet'],
+    async (ctx) => {
+      await ctx.run(`npx pilet build --target './different-target/index.js'`);
+
+      await ctx.assertFiles({
+        dist: false,
+        'different-target/index.js'(content: string) {
+          expect(content).not.toBe('');
+          expect(content).toContain('//@pilet v:2');
+          expect(content).toContain('System.register(');
+          expect(content).not.toContain('currentScript.app=');
+          expect(content).not.toContain('require("react")');
+        },
+        'different-target/main.css': true,
+        'dist/index.js': false,
+      });
+    },
+  );
+
+  // 'can build a pilet with minify --done'
+  test('from-sample-piral-with-minify', 'can build a pilet with minify', ['pilet.v2', 'build.pilet'], async (ctx) => {
+    await ctx.run(`npx pilet build --minify`);
+
+    await ctx.assertFiles({
+      'dist/index.js'(content: string) {
+        expect(content).not.toBe('');
+        expect(content).not.toContain('var moduleUrl');
+      },
+    });
+  });
+
+  // 'can build a pilet without minify' --done,
+  test(
+    'from-sample-piral-without-minify',
+    'can build a pilet without minify',
+    ['pilet.v2', 'build.pilet'],
+    async (ctx) => {
+      await ctx.run(`npx pilet build --no-minify`);
+
+      await ctx.assertFiles({
+        'dist/index.js'(content: string) {
+          expect(content).not.toBe('');
+          expect(content).toContain('var moduleUrl');
+        },
+      });
+    },
+  );
+
+  // 'can build a pilet with standalone type' --done,
+  test(
+    'build-pilet-with-standalone-type',
+    'can build a pilet with standalone type',
+    ['pilet.v2', 'build.pilet'],
+    async (ctx) => {
+      await ctx.run(`npx pilet build --type 'standalone'`);
+
+      await ctx.assertFiles({
+        'dist/standalone/index.html'(content: string) {
+          expect(content).toMatch(/src="\/[a-zA-Z0-9\.\-\_]*?\.js/g);
```

Comment:
Ok but the pilet should have also been build in the standalone mode. I think a few more checks are necessary here, too (besides the `dist/standalone/index.html`).